### PR TITLE
update stack resolver to lts-10.3

### PR DESCRIPTION
**Pull request by matthewleon**
`stack build` will thus use GHC 8.2.2

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
-resolver: nightly-2017-11-20
+resolver: lts-10.3
 packages:
 - '.'
 extra-deps:
 - pipes-http-1.0.5
 - Win32-notify-0.3.0.3
```
Comment (kRITZCREEK) [inline stack.yaml]: this seems to be in the resolver now.

File: stack.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+- ansi-terminal-0.6.3.1
```
Comment (kRITZCREEK) [inline stack.yaml]: why can't we bump this to the resolvers 0.7.1.1 ?

Comment (matthewleon) [reply stack.yaml]: I reviewed the changes and I don't see any reason we can't. I was being too conservative.

**Comment by matthewleon** [comment]
No idea what to make of the fail on AppVeyor.

**Comment by matthewleon** [comment]
possibly need to upgrade stack? https://github.com/fpco/stackage/issues/2759

**Comment by kRITZCREEK** [comment]
You could try and bump the stack version in appveyor.yml to 1.6.3. Not sure how this would work for travis

**Comment by matthewleon** [comment]
> You could try and bump the stack version in appveyor.yml to 1.6.3.

Ah, I stupidly missed that file. Thanks.

**Comment by kRITZCREEK** [comment]
Nothing stupid there, I had to look it up myself.

**Comment by hdgarrood** [comment]
Thanks! I decided to just go ahead and merge since it was just macOS builds we were waiting on.

**Pull request merged**